PPLUS-331: Updated readme (#10)
spryker-sdk/upgrader: README.md

It is recommended that you install Upgrader globally:

```
composer global require spryker-sdk/upgrader
```

If you want it in your project go into your project folder and run:

```
composer require spryker-sdk/upgrader --dev
```

## Documentation

### Configuration

Prepare required environment variables:

```
export GITHUB_ACCESS_TOKEN=<GITHUB_TOCKEN> # GitHub access token to project repository with permission to push branches and create PRs.
export GITHUB_ORGANIZATION=<ORGANIZATION> # Free string.
export GITHUB_REPOSITORY=<REPOSITORY> # Free string.
```

### Commands

Upgrade command:

> 1. Performs check that "is exists uncommitted changes in a project", if exist - the process will stop.

Reviewer: ```suggestion
1. Performs check that "there are no uncommitted changes in a project". If they exist, the process will stop.
```

> 2. Upgrades Spryker packages.

Author: Updated.

3. Commits there changes in composer.json and composer lock files.

> 4. Push changes and creates PR.

Reviewer: ```suggestion
4. Pushes changes and creates PR.
```
consistent usage of 3rd person language.
